lib/logstash/outputs/elasticsearch/buffer.rb

```diff
@@ -1,124 +0,0 @@
-require 'concurrent'
```

Comment:
Ah, I follow - we are using the batches from pipeline so there's no need for `Stud::Buffer`